src/prg_system_mod.F90

```diff
@@ -254,17 +257,17 @@ subroutine prg_parse_system(system,filename,extin)
 
     if(.not.present(extin))then
        call prg_get_nameandext(filename,nametmp,extension)
-       filename = nametmp
     else
        extension = extin
+       nametmp = trim(filename)
```

Comment:
You could also `trim(adjustl(filename))` to remove potentially leading whitespaces.

Reply:
Changed